webapp/app/plugins/tabs.js

```diff
@@ -34,11 +34,11 @@ treeherder.factory('thTabs', [
                     enabled: false
                 }
             },
-            "selectedTab": "failureSummary",
+            "selectedTab": "jobDetails",
             "showTab" : function(tab, contentId){
                 thTabs.selectedTab = tab;
                 if(!thTabs.tabs[thTabs.selectedTab].enabled){
-                    thTabs.selectedTab = 'failureSummary';
+                    thTabs.selectedTab = 'jobDetails';
```

Comment:
This and line 37 are up for debate, they seem to be fine to switched to be the default (doesn't seem to produce any terrible annoying refresh-switch when loading a failed job which then loads Failure summary), but I defer to @camd.